### PR TITLE
Media section: Action bar buttons + card

### PR DESCRIPTION
**Pull request by artpi**
Closes task #10805
- Restyles buttons into more compact ones and uniform ones
- Removes secondary actions from the bottom of modal view
- Removes Disk space usage from secondary actions component

## Show me

zrzut ekranu 2017-02-28 o 14 35 00

zrzut ekranu 2017-02-28 o 14 34 44

CC @iamtakashi @rralian @gwwar

### CODE REVIEW

**Comment by gwwar** [comment]
@iamtakashi aws is still out so I can't attach a picture yet, but I'm thinking clicking on the chevron should give us a select dropdown instead of a popover link.

Also how would we feel about adding label text to the delete button?

File: client/post-editor/media-modal/secondary-actions.jsx [hunk 1]
```diff
@@ -73,8 +54,9 @@ const MediaModalSecondaryActions = React.createClass( {
 		if ( ModalViews.LIST === view && selectedItems.length ) {
 			buttons.push( {
 				key: 'edit',
-				value: this.translate( 'Edit' ),
+				text: this.translate( 'Edit' ),
```
Comment (gwwar) [inline client/post-editor/media-modal/secondary-actions.jsx]: In the media modal view, the primary action on top is edit, while the bottom primary action is continue. @artpi @iamtakashi  should we try to match this?

File: client/post-editor/media-modal/secondary-actions.jsx [hunk 2]
```diff
@@ -96,89 +78,18 @@ const MediaModalSecondaryActions = React.createClass( {
...
-		return this.getButtons().map( ( button ) => {
-			return React.createElement( 'input', Object.assign( {
-				type: 'button'
-			}, button, {
-				className: classNames( 'editor-media-modal__secondary-action', 'button', 'is-desktop', button.className )
-			} ) );
-		} );
-	},
-
-	renderPlanStorage() {
```
Comment (gwwar) [inline client/post-editor/media-modal/secondary-actions.jsx]: I believe we'll need this 😄  As is we won't ever see this component. Let's wait for #11639 to land, and we can rebase both this branch and the target branch.

Comment (artpi) [inline client/post-editor/media-modal/secondary-actions.jsx]: Rebased :)

**Comment by artpi** [comment]
> I'm thinking the add url action should be a dropdown, or at least match the compact styling much more. Current line spacing makes this look really odd.

I made compact styling there:
zrzut ekranu 2017-03-01 o 08 55 38

**Comment by artpi** [comment]
I'om going to merge this in a moment and make issues for remaiming tasks:

>  Bug with edit/delete button displaying on top action bar. In the "all" tab select some images, click on "video". The edit/delete items still display

This is a bug currently in modal as well

> Making sure plan storage appears

Im gonna make an issue for it

**Comment by artpi** [comment]
> Bug with edit/delete button displaying on top action bar. In the "all" tab select some images, click on "video". The edit/delete items still display

https://github.com/Automattic/wp-calypso/issues/11665

> Making sure plan storage appears

https://github.com/Automattic/wp-calypso/issues/11667

**Comment by gwwar** [comment]
The dropdown still looks a bit weird in https://github.com/Automattic/wp-calypso/pull/11560#issuecomment-283270797 . Maybe we could go with something similar to the insert dropdown. cc @iamtakashi 

screen shot 2017-03-01 at 10 02 01 am

**Comment by iamtakashi** [comment]
I think we can remove the minimum width for this popover.

image

> Also how would we feel about adding label text to the delete button?

If we add a label to the delete button, the edit button will beg to have an icon too. That's ok, but we have many things going on the action bar, and I wanted to avoid a situation where we need to hide and show labels and icons depending on viewport sizes. Having text label almost always help to clarify icons though, the bin/trash is the safest bet to remove the label to save some space.

image

I've noticed some spacing between buttons don't seem the same.

**Pull request merged**